lib/environments/meteorEnvironment.js

```diff
@@ -1,22 +1,46 @@
+import fs from 'fs';
+import path from 'path';
+
+import findPackageDependencies from '../findPackageDependencies';
+
+const coreModules = [
+  'meteor/accounts-base',
+  'meteor/blaze',
+  'meteor/check',
+  'meteor/ddp-client',
+  'meteor/ddp-rate-limiter',
+  'meteor/ejson',
+  'meteor/email',
+  'meteor/http',
+  'meteor/check',
+  'meteor/meteor',
+  'meteor/mongo',
+  'meteor/random',
+  'meteor/reactive-var',
+  'meteor/session',
+  'meteor/templating',
+  'meteor/tracker',
+];
+
+function meteorPackageDependencies(
+  { config }: Object
+): Array<string> {
+  const meteorPackagesPath = path.join(config.workingDirectory, '.meteor/packages');
```

Comment:
I'm looking for the same. May have to just create it and place it here in a comment. "packages" is considered to be an internal file though most users maintain it by hand instead of using the provided `meteor add`, `meteor remove` and `meteor list` commands. It implicitly supports such usage by allowing comments (prefixed with "#") and blank lines.